bndtools.core/src/bndtools/command/BndResolveHandler.java

```diff
@@ -37,13 +37,13 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 			public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
 				IStatus status = Status.OK_STATUS;
 				try {
-					Bndrun bndrun = Bndrun.createBndrun(Central.getWorkspace(), new File(bndrunFile.getLocationURI()));
-					Project bndProject = Central.getProject(project);
-					bndrun.setBase(bndProject.getBase());
+					Bndrun bndrun = (Bndrun) LaunchUtils.createRun(bndrunFile, RunMode.LAUNCH);
 
 					bndrun.resolve(false, true);
 
-					bndrunFile.refreshLocal(IResource.DEPTH_ONE, monitor);
+					if (Central.isBndProject(project)) {
```

Comment:
Hey @rotty3000   Thanks for wiring this up with Maven support! 

Why did you want to predicate this on being a bndProject.  Will that match maven projects?

Reply:
Because in M2E these files are already being monitored and would result in an infinite loop if I added this other propagation.